dpgen/generator/run.py

```diff
@@ -2483,18 +2534,22 @@ def post_fp_gaussian (iter_index,
         sys_output = glob.glob(os.path.join(work_path, "task.%s.*/output"%ss))
         sys_output.sort()
         for idx,oo in enumerate(sys_output) :
-            sys = dpdata.LabeledSystem(oo, fmt = 'gaussian/log')
-            if len(sys) > 0:
-                sys.check_type_map(type_map = jdata['type_map'])
-            if jdata.get('use_atom_pref', False):
-                sys.data['atom_pref'] = np.load(os.path.join(os.path.dirname(oo), "atom_pref.npy"))
-            if idx == 0:
-                if jdata.get('use_clusters', False):
-                    all_sys = dpdata.MultiSystems(sys, type_map = jdata['type_map'])
+            # TODO : UnboundLocalError sometimes occurs and I cannot figure it out.
```

Comment:
I don't think UnboundLocalError is an expected error that should be caught. If it happened, it is a bug that needs to be fixed.

Reply:
Sure. I met this problem this week but I don't have time to fix it :(
I will remove this part and try to fix it afterwards.